src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -0,0 +1,83 @@
+package mate.academy.bookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+
+@Entity
```

Comment:
```suggestion
@Entity
@Getter
@Setter
```
You can use Lombok annotations to generate get and set methods